src/Autofac/Features/ImplicitRegistrationSource.cs

```diff
@@ -0,0 +1,110 @@
+﻿// This software is part of the Autofac IoC container
```

Comment:
We don't currently have any types directly under Features; I think this class needs to go under Autofac.Core, at the same level as `IRegistrationSource`.

Reply:
Done